test/functional/controller/controller_suite_test.go

```diff
@@ -1,6 +1,6 @@
 // +build unit_test
```

Comment:
is this still applicable after the test has moved to _functional/_?

Reply:
Probably not,  they were useful early on but all build tags should probably be dropped until we come up with a standard for them.